qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -254,6 +255,72 @@ def defaults(self, refresh: bool = False) -> Optional[PulseDefaults]:
 
         return self._defaults
 
+    def job_limit(self) -> BackendJobLimit:
+        """Return job limit for the backend.
+
+        The job limit information may include the current number of
+        jobs running on the backend at this time and the maximum number
+        of concurrent jobs a user could submit to the backend at a time.
```

Comment:
I think the first part of the sentence needs to indicate it's the user's jobs, since it may be misinterpreted as all jobs. Maybe something like 
`The job limit information may include, for this backend, the current number of unfinished jobs you have and the maximum number of unfinished jobs you can have.`